facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java

```diff
@@ -302,4 +337,65 @@ public void setDependentArtifacts(int id, List<Integer> artifactIds) {
         buildRecord.setDependencies(artifacts);
         repository.save(buildRecord);
     }
+
+    public Page<BuildPushResult> brewPush(BuildPushRequest buildPushRequest, String callbackUrl) throws ProcessException {
```

Comment:
BTW there is https://github.com/project-ncl/pnc/blob/master/facade/src/main/java/org/jboss/pnc/facade/impl/BrewPusherImpl.java
I would implement this stuff there.

Reply:
Ack! Moved to BrewPusher Impl